ministryofjustice/hmpps-incident-reporting
# IR-496: Download DPS & NOMIS config as JSON

```mermaid
flowchart TD
    Download[Download DPS config JSON via UI app] --> Import[Import files using CLI from UI app locally]
    Import --> Git[Check updated enumerations and constants into git]
    Git --> End([ ])
```

Steps to get latest incident reporting API constants imported:
- download DPS constants/enumerations
- run script to import them into generated typescript files
- check into git
- latest types from incident reporting api are now baked into this repo

Steps to get latest NOMIS configuration imported:
- download NOMIS JSON files
- [NOT IMPLEMENTED] run script to import them into generated typescript files
- [NOT IMPLEMENTED] check into git
- [NOT IMPLEMENTED] latest configuration from NOMIS are now baked into this repo

As noted in the README, download links are listed using `./scripts/listDownloadLinks.ts prod` command.
